fluent-bit/templates/daemonset.yaml

```diff
@@ -21,6 +21,9 @@ spec:
       imagePullSecrets:
         {{- toYaml . | nindent 8 }}
       {{- end }}
+      {{- with .Values.priorityClassName }}
+      priorityClassName: {{ . }}
+      {{- end }}
```

Comment:
https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.15/#podspec-v1-core
It's good. I checked that it belongs to PodSpec.